include/package-pack.mk

```diff
@@ -344,7 +344,7 @@ else
 	  --info "name:$(1)$$(ABIV_$(1))" \
 	  --info "version:$(VERSION)" \
 	  --info "description:$$(call description_escape,$$(strip $$(Package/$(1)/description)))" \
-	  --info "arch:$(PKGARCH)" \
+	  $(if $(findstring all,$(PKGARCH)),--info "arch:noarch",--info "arch:$(PKGARCH)") \
```

Comment:
Is this still needed?

Reply:
Yeah, I think so cause I see APK updating the packages that have arch set to `all` when installing a new package but when its set to `noarch` then it doesn't do it.

If I understand the upstream APK commit we are supposed to use `noarch` anyway